other/materials_designer/create_slab.ipynb

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
<p>This is not clear, we should refer to README to explain what happens in Pyodide and Python.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>